Library/Formula/agda.rb

```diff
@@ -17,29 +15,37 @@ class Agda < Formula
     sha256 "d0e17808ee896cdedc499b010b67961dc3db3c430c501faef16c4b4482238447" => :mountain_lion
   end
 
-  head "https://github.com/agda/agda.git", :branch => "master"
+  head do
+    url "https://github.com/agda/agda.git", :branch => "master"
+    resource "stdlib" do
+      url "https://github.com/agda/agda-stdlib.git", :branch => "master"
+    end
+  end
+
+  resource "stdlib" do
+    url "https://github.com/agda/agda-stdlib.git",
+        :tag => "v0.11",
+        :revision => "8602c29a7627eb001344cf50e6b74f880fb6bf18"
+  end
 
   option "without-stdlib", "Don't install the Agda standard library"
-  option "with-malonzo-ffi",
-    "Include the MAlonzo backend's FFI (depends on the standard library)"
+  option "without-malonzo", "Disable the MAlonzo backend"
 
-  depends_on "ghc" => :build
+  if build.with? "malonzo"
+    depends_on "ghc"
+  else
+    depends_on "ghc" => :build
+  end
   depends_on "cabal-install" => :build
 
   depends_on "gmp"
-  depends_on "emacs" => :optional
+  depends_on "emacs" => :recommended
```

Comment:
While we're at it, this can be switched to `depends_on :emacs`, and if there's a minimum version that can be specified too with
```ruby
depends_on "emacs" => ["24.1", :recommended]